src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -100,30 +99,29 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
             ${supportsFeature(stateObj, SUPPORT_BRIGHTNESS)
               ? html`
                   <round-slider
-                    min="1"
+                    min="0"
```

Comment:
This changes the function of the slider.
Now sliding to the bottom will turn the light off as opposed to setting it to minimum brightness.
This results in the handle disappearing, which might be confusing.

Reply:
This fixes the handle disappearing, but yeah it does turn the light off when sliding all the way down. I think I prefer it this way. Do we have more options? Old behaviour was not turning off but still showing the handle?

Reply:
Min was 1 with the old slider too. https://github.com/home-assistant/home-assistant-polymer/pull/3634/files#diff-f2231026bfc2c43209404df9b7bbb679L35

It's probably better this way. I just didn't want it to slide by without comment.

Reply:
But in the old version the handle did stay, is that something that was changed in `round-slider`?

Reply:
Probably. The slider is disabled if the value is outside of the range.
Maybe the old one drew it outside the bar, or just clamped it to the min-max range?

Reply:
I'm not a fan of slider turning off the light. Getting to 1% brightness is going to take skill now

Reply:
I suggest to make this change now, so we have the handle back when the light is off, and check if we can get the old behaviour of round slider back.